code/web/src/modules/subscription/api/actions.js

```diff
@@ -29,7 +30,7 @@ export function getList(isLoading = true) {
 
     return axios.post(routeApi, query({
       operation: 'subscriptions',
-      fields: ['id', 'user { name, email }', 'crate { id, name, description }', 'createdAt']
+      fields: ['id', 'date', 'user { name, email }', 'crate { id, name, description }', 'createdAt']
```

Comment:
Thanks again back end! Y'all made this super simple for us!

Reply:
So glad to hear that!